backend/app/views/spree/admin/orders/index.html.erb

```diff
@@ -7,7 +7,7 @@
   <li>
     <%= link_to t('spree.new_order'), new_admin_order_url, id: 'admin_new_order', class: 'btn btn-primary' %>
   </li>
-<% end if can? :create, Spree::Order %>
+<% end if can? :manage, Spree::Order %>
```

Comment:
why ``manage` if the link is for creating orders only?

Reply:
Hello @kennyadsl . 

There are two classes to manage the permissions sets Spree::PermissionSets::OrderManagement and Spree::PermissionSets::OrderDisplay. The OrderManagement establishes the ':manage' permissions. The display establishes the ':display' and ':admin' permissions. If I want to restrict a user with only the permission OrderDisplay, I have to check for ':manage' permission as for some reasons, the ':create' permission on OrderDisplay is still allowing the button to be rendered, allowing the user to click the New Order button, and creating an empty order that it cannot be modified, not great user experience I guess. I'm not sure why the ':create' is still available on OrderDisplay. Do you know how to fix this issue? Maybe I misunderstood the OrderDisplay role. Let me know your thoughts about it.

Reply:
Hello @kennyadsl 

Do you have any thoughts about my previous comments on PermissionSets? 

Regards

Reply:
By default only [this permissions set](https://github.com/solidusio/solidus/blob/master/core/lib/spree/permission_sets/default_customer.rb) is activated. I think that at line 10 it sets the create ability for Order. Are you setting the other permissions for a specific user in some way via code?

Reply:
From what I can understand from @memotoro's comment and the proposed patch is that, `OrderDisplay` works **only** to create orders without any further actions, whereas `OrderManagement` allows full control over them. Since we're only talking about `backend`, this feels reasonable.

As @tvdeyen said on his comment below, if you can create an order, you probably should be able to update it as well.